keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -887,6 +916,9 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+Same as above: all the components must have support for it to work.
+Specifically, every component will ignore or filter the field if it doesn't recognize/support it.
```

Comment:
Please expand on behavior when the kubelet is at a level where this isn't enabled by default.